install.sh

```diff
@@ -20,9 +11,4 @@ if command_exists python2; then
   PYTHON_BINARY=python2
 fi
 
```

Comment:
Consider echoing a message that `install.sh` is being kept around for backwards compatibility and that people should be calling `install.py` directly.